pkg/cluster/internal/kubeadm/config.go

```diff
@@ -90,6 +90,13 @@ apiServerExtraVolumes:
 # so we need to ensure the cert is valid for localhost so we can talk
 # to the cluster after rewriting the kubeconfig to point to localhost
 apiServerCertSANs: [localhost]
+kubeletConfiguration:
+  baseConfig:
```

Comment:
I _think_ this is right for the older types... https://github.com/kubernetes/kubernetes/blob/e6f6fa1f2dd136552e0d6e507b943e5adeaeae09/cmd/kubeadm/app/apis/kubeadm/v1alpha2/types.go#L286 @neolit123

Reply:
yes